packages/bibtex/styles/chicago.lua

```diff
@@ -1,5 +1,37 @@
 local Bibliography = require("packages.bibtex.bibliography")
 
+-- WORKAROUND FIXME
+-- We may want fluent strings for all languages, but for now we just use English.
```

Comment:
See #2074 :
It would be a shame to have all possible date composition rules in our ftl files, while CSL locales have them.
(Which doesn't imply that fluent would necessarily be out of the question -- Just that the input should be a CSL locale, however managed afterwards)

Reply:
But also not that CSL styles can override some of the CSL locales.
So proper handling of such rules = CSL locales + CSL styles.

Reply:
In-code comment improved.